labelbox/schema/bulk_import_request.py

```diff
@@ -585,27 +606,52 @@ class DataRow(BaseModel):
 
 
 class NDFeatureSchema(BaseModel):
-    schemaId: str = LabelboxID
+    schemaId: Optional[Cuid] = None
+    name: Optional[str] = None
+
+    @root_validator
+    def must_set_one(cls, values):
+        if values['schemaId'] is None and values['name'] is None:
+            raise ValueError(
+                "Must set either schemaId or name for all feature schemas")
+        return values
 
 
 class NDBase(NDFeatureSchema):
     ontology_type: str
     uuid: UUID
     dataRow: DataRow
 
-    def validate_feature_schemas(self, valid_feature_schemas):
-        if self.schemaId not in valid_feature_schemas:
-            raise ValueError(
-                f"schema id {self.schemaId} is not valid for the provided project's ontology."
-            )
+    def validate_feature_schemas(self, valid_feature_schemas_by_id,
+                                 valid_feature_schemas_by_name):
+        if self.name:
+            if self.name not in valid_feature_schemas_by_name:
+                raise ValueError(
+                    f"name {self.name} is not valid for the provided project's ontology."
```

Comment:
Let's make sure the error outputted to customers are all formatted and capitalized consistently.
`Name {self.name} is not valid for the provided project's ontology`